modules/thermal_hydraulics/test/tests/components/inlet_mass_flow_rate_1phase/jac.massflowrate_3eqn_water97.i

```diff
@@ -0,0 +1,87 @@
+[GlobalParams]
```

Comment:
What does this test test that the existing one does not, in regards to `InletMassFlowRateTemperature1Phase`?

Reply:
We need to figure out a way to test the fluid property compatibility with THM. For example, there is a test for the calorically imperfect gas in the VolumeJunction1Phase tests because the lookups in that component were causing problems at some point.

Reply:
Maybe we should make a simple yet maximal input file for THM and NS and place in a special test folder like navier_stokes/test/tests/fluid_prop_compatibility (same for thm). 

Then every time we add a FP we add an inactive (only active for one test) block in that input file.

Reply:
Yeah, we could do something like that. I won't require it in this PR.

As for the tagging system, that sounds nice, but may be a lot of work, and has the potential to get complicated (like if there are multiple flow models in THM/NS).

Reply:
Yes I just wanted to be able to run a THM problem with `Water97FluidProperties` ... I did not realize I would only get to that point almost 3k lines later :grimacing: :laughing:

Reply:
I personally think that most fluid properties classes should go in the module that uses them ... then at least you would instantly know one module that the FP object is compatible with ... and then it's a little bit easier to get coverage testing as well. But I also agree that a table in documentation would be great

Reply:
I ll add the table soon then

Reply:
I like to just run it and see if it works - it makes it more exciting. Also, that list could be a little restrictive if for example, some obscure interface is used by some component that is rarely used. Maybe if you're going to make a table, just mention that possibility - that they can just try it, even though the table says something else.